src/main/scala/cromwell/engine/ExecutionStore.scala

```diff
@@ -27,10 +27,13 @@ class ExecutionStore(binding: WdlBinding) {
     for {
       callEntry <- table if callEntry._2 == ExecutionStatus.NotStarted
       call = callEntry._1
-      if call.prerequisiteCalls().forall(table.get(_).get == ExecutionStatus.Done)
+      if call.prerequisiteCalls().forall {
```

Comment:
prerequeisiteCalls returns Iterable[Call] - why would something not be of type call?